app/services/national_scoreboard.rb

```diff
@@ -21,12 +21,16 @@ def to_param
     SLUG
   end
 
-  def this_academic_year(today: Time.zone.today)
-    start_year = today.year - (today.month < 9 ? 1 : 0)
+  def academic_year_for(date:)
```

Comment:
We've got a gap in our test coverage as these methods are used in SchoolGroup, see `this_academic_year` and `previous_academic_year` which are overidden to deal with schools without a scoreboard or a default calendar. So will break some group scoreboards.